File: python/Sumitpython/discount.py
amt=int(input("enter amt from 1000 to 10000"))
#amt=float
if(amt>=1000 and amt<=5000):        
    amt=amt-amt*10/100
elif(amt>=5100 and amt<=10000):
    amt=amt-amt*20/100
else:
    amt>10000
    amt=("amt is not apropriate")
print("This is your net amount",amt)

File: C&C++/Rahul/discount.c
#include<stdio.h>
int main()
{
    int age;
    float discount,net_amount,amount;
    printf("Enter your age :- ");
    scanf("%d",&age);
    printf("***********************************************************\n");
    printf("Enter your amount :- ");
    scanf("%f",&amount);
    printf("***********************************************************\n");
    if(age > 50)
    {
        discount=amount*0.10;
        printf("Your discount is %f\n",discount);
    }
    else if(age <= 50 && age >= 14 )
    {
        discount=amount*0.20;
        printf("Your discount is %f\n",discount);
    }
    else if(age<14)
    {
       discount=amount*0.50;
        printf("your discout is %f\n",discount);
    }
    else
    {
        printf("you entered a wrong value\n");
    }
    printf("***********************************************************\n");
    net_amount=amount-discount;
    printf("Your total amount is %f with discount\n",net_amount);
    printf("***********************************************************\n");
}